currency-converter/currency.js

```diff
@@ -0,0 +1,75 @@
+document.addEventListener("DOMContentLoaded", () => {
+  const rates = {
+    timestamp: Date.now(),
+    base: "EUR",
+    date: new Date().toISOString().split("T")[0],
+    rates: {
+      USD: 1.23,
+      GBP: 0.88,
+      EUR: 1.82,
+    },
+  };
+
+  const insertRateForm = document.getElementById("insertRateForm");
+  const convertForm = document.getElementById("convertForm");
+  const updateRateForm = document.getElementById("updateRateForm");
+  const conversionResult = document.getElementById("conversionResult");
+
+  insertRateForm.addEventListener("submit", (event) => {
+    event.preventDefault();
+    const baseCurrency = document
+      .getElementById("baseCurrency")
+      .value.toUpperCase();
```

Comment:
Good idea to clean up the user input! You should `trim()` it too

Reply:
sure, will do that.